src/index.html

```diff
@@ -1,19 +1,597 @@
 <!doctype html>
-<html lang="en">
+<html
+  lang="en"
+  class="page"
+>
   <head>
     <meta charset="UTF-8" />
+
     <meta
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
-    <title>Title</title>
+
+    <title>NAMU</title>
+
+    <link
+      rel="icon"
+      type="image/x-icon"
+      href="/src/images/icons/fav-icon.png"
+    />
+
     <link
       rel="stylesheet"
       href="./styles/main.scss"
     />
   </head>
-  <body>
-    <h1>Hello Mate Academy</h1>
+  <body class="page__body">
+    <div class="header">
+      <div class="container container--header">
+        <div class="top-bar header__top-bar">
+          <a href="#">
+            <img
+              src="./images/logo.svg"
+              alt="NAMU logo"
+              class="top-bar__logo"
+            />
+          </a>
+
+          <div class="top-bar__icons">
+            <a
+              href="#menu"
+              class="icon icon--menu top-bar__icon"
+            ></a>
```

Comment:
you can add `aria-label ` to icons for better screen reader support.
```suggestion
            <a
              href="#menu"
              class="icon icon--menu top-bar__icon"
              aria-label="Open menu"
            ></a>
```